src/runtime/hexagon_dma.cpp

```diff
@@ -601,14 +606,14 @@ WEAK int halide_hexagon_dma_device_sync(void *user_context, struct halide_buffer
         << "Hexagon: halide_hexagon_dma_device_sync (user_context: " << user_context
         << " buf: " << buf << ")\n";
 
-    dma_device_handle *dev = (dma_device_handle *)buf->device;
-    halide_assert(user_context, dev->dma_engine);
+ /* dma_device_handle *dev = (dma_device_handle *)buf->device;
```

Comment:
TODO on this one too?

Reply:
Added TODO comment . Not sure what to do in this function right now